tools/gulp/build.js

```diff
@@ -29,7 +29,8 @@ module.exports = (gulp, shared) => {
           `packages/${pkg}/src/**/*.{js,jsx}`,
           `!packages/${pkg}/src/**/{__mocks__,__tests__}/*.{js,jsx}`,
           `!packages/${pkg}/src/**/*.example.{js,jsx}`,
-          `!packages/${pkg}/src/**/*.test.{js,jsx}`
+          `!packages/${pkg}/src/**/*.test.{js,jsx}`,
+          `!packages/${pkg}/src/helpers/**/*.{js,jsx}`
```

Comment:
Files from `helpers/e2e` were being included in the build, this removes them because the code was related to testing. 

I wasn't entirely sure if I should have ignored only `e2e` or the entire `helpers` folder. I.e., if there's code in helpers and that's being used by a component, does it need to be included here or is it automatically included?

Reply:
I think since we've labeled the `helpers/e2e` folder as an _e2e_ thing by calling it that, we're saying that that part is for e2e tests and the rest of `helpers` is for whatever else. That is, we're not prescribing a use for all of `helpers/` but we are for `helpers/e2e/`, so maybe we should only ignore that part. It's possible that we put other things in there in the future.

Reply:
It would also be possible to overload our use of the `utilities` folder by putting javascript in there, but that could get messy. What do you think?

Reply:
I'm fine with just only ignoring `helpers/e2e` if that's ok with you!